packages/react-jss/src/createHoc.js

```diff
@@ -183,23 +183,21 @@ export default function createHOC<
       return staticSheet
     }
 
-    getDynamicSheet(staticSheet) {
+    getDynamicSheet(staticSheet: StyleSheet): StyleSheet | void {
       // $FlowFixMe Cannot access random fields on instance of class StyleSheet
       const dynamicStyles = staticSheet[dynamicStylesNs]
 
-      if (dynamicStyles) {
-        const contextSheetOptions = this.context[ns.sheetOptions]
+      if (!dynamicStyles) return undefined
```

Comment:
this could just be `return;` right?

Reply:
yeah, we need to change our eslint config for this, it requires explicite return value right now

Reply:
Ah ok, yeah makes sense to enforce explicit return value.